test/util.sh

```diff
@@ -11,9 +11,11 @@ export MGMT="$ROOT/mgmt"
 if [[ $(uname) == "Darwin" ]] ; then
 	export timeout="gtimeout"
 	export mktemp="gmktemp"
+	export STAT="gstat"
```

Comment:
how come some of these are uppercase, and others are lowercase? what is correct?

Reply:
If it is not capitalised it hits: https://github.com/koalaman/shellcheck/wiki/SC2154

Preferably all shell scripts should be conform shellcheck but thats a branch I still need to finish.